scripts/setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
You can add the `set -e` command to stop the setup script if any of the commands fail.  Normally it'd be possible to add the `-e` directly to the `#!/bin/bash` shebang, but that doesn't seem to work with `#!/usr/bin/env bash`.  I do believe using `/usr/bin/env` is probably the most compatible here, having now dealt with systems that keep it in /usr/local and unpredictable places (nixos).  :)